en/editor/extension/define.md

```diff
@@ -26,42 +26,42 @@ The extension package needs to be pre-defined with some basic information that n
 
 ## name
 
-Type {string} Required
+Type: `{string}` Required
 
 The name of the extension, this name needs to correspond to the extension folder one-to-one.
 
 ## version
 
-Type {string} Required
+Type `{string}` Required
 
 The version number of the extension is used to submit the version verification of the extension, as well as some upgrades of the extension itself, and the data migration is used as the basis for comparison.
 
 ## author
 
-Type {string} Optional
+Type `{string}` Optional
 
-The name of the extension author will be displayed in the "Extension Manager".
+The name of the extension author will be displayed in the **Extension Manager**.
 
 ## description
 
-Type {string} Optional
+Type `{string}` Optional
 
-The description of the extension, briefly summarize the function of the extension. Support i18n:key grammar.
+The description of the extension, briefly summarize the function of the extension. Support `i18n:key` grammar.
 
 ## main
 
-Type {string} Optional
+Type `{string}` Optional
 
-The relative path of a js file defines the function entry file. When the extension starts, the js file pointed to by the main field will be executed, and the corresponding method will be triggered or executed according to the process.
+The relative path of a js file defines the function entry file. When the extension starts, the JavaScript file pointed to by the main field will be executed, and the corresponding method will be triggered or executed according to the process.
 
 ## panels
 
-Type {[name: string]: PanelInfo} Optional
+Type `{[name: string]: PanelInfo}` Optional
 
-Panel information defined in the extension. You can use Editor.Panel.open('hello-world.list'); to open the defined panel. For more information, please refer to [Panel](./panel.md)
+Panel information defined in the extension. You can use `Editor.Panel.open('hello-world.list');` to open the defined panel. For more information, please refer to the [Panel](./panel.md) documentation.
 
 ## contributes
 
-Type {[name: string]: any} Optional
+Type `{[name: string]: any}` Optional
 
-Extend existing functions, and customize some other function modules that are open to the outside world. For more information, please refer to [Contributions](editor/extension/contributions.md)
+Extend existing functions, and customize some other function modules that are open to the outside world. For more information, please refer to the [Contributions](editor/extension/contributions.md) documentation.
```

Comment:
\[Contributions]\(./contributions.md)

Reply:
the Chinese doc is wrong also, thus why English is wrong.